packages/@lightningjs/ui-components/src/components/TextBox/TextBox.js

```diff
@@ -271,6 +273,14 @@ export default class TextBox extends Base {
     }
   }
 
+  _getMarqueeProps() {
+    return this._marqueeProps ?? {};
+  }
+
+  _setMarqueeProps(props) {
+    this._marqueeProps = props;
```

Comment:
if all this isn't mutating the value or handling a side effect, it can be omitted. the way the `_set<PropName>` methods work is they get called at the beginning of the setter method, process the value, and then pass it back to the setter. One added benefit is we don't need matching pairs like we do if we were overriding the accessor methods